src/DataAPI.jl

```diff
@@ -183,4 +183,102 @@ default definition.
 function unwrap end
 unwrap(x) = x
 
+"""
+    innerjoin(x, y, zs...)
```

Comment:
Two small comments:
1. Here and below I would drop `zs`, as other packages do not have to provide this.
2. I think it would be good to mention a reference (or add a description) to what a given type of join does. Ideally (if it is possible to do it relatively easily and in a generic way) - maybe also add a comment that the way key columns are determined is implementation specific and that key equality rules are also implementation specific.

Reply:
Actually I'm not sure it's very useful to provide docstrings. Packages will have to provide more specific docstrings anyway, and the default docstrings are even misleading as `on` may be required by some packages.

For `pairwise` in StatsAPI I just added a comment for the same reason. I think that makes sense since that documentation is intended at package authors rather than end users.

Reply:
I am OK to fully remove docstrings.

Reply:
I remove the docstrings and add some code comments, since it's face to developers.